src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -7646,6 +7671,7 @@ export class StudyViewPageStore
         }
     >(
         q => ({
+            await: () => [this.filtersRemote],
```

Comment:
@alisman I don't know if there is a better way to do this kind of thing. I referred to me and @BasLee 's conversation below:

> Me: We wait for `filtersRemote` but we don't use it in `invoke`, is that because you want to trigger the `filtersRemote` to fetch data? 

> Bas: Yes, I added this because we need `this.filters` to be completely loaded, which in turn relies on the async `this.selectedDriverTiersMap`

Reply:
lets discuss on tuesda

Reply:
This problem has been fixed after rebasing to master, see also [this comment](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/files/2006e93283a4ab1f98f1986685dc7c1c4b1269cc#r1168362636)